purchase_manual_delivery/__manifest__.py

```diff
@@ -8,7 +8,7 @@
         and adds the ability to manually generate them as the supplier confirms
         the different purchase order lines.
     """,
-    "version": "16.0.1.1.4",
+    "version": "16.0.1.1.5",
```

Comment:
Next time, you don't have to update the version number yourself. Ocabot will do the honours.